src/main/cpp/commands/ShooterCommand.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "commands/ShooterCommand.h"
+
+ShooterCommand::ShooterCommand(Shooter& shooter,
+                                     ControlBoard& humanControl)
+    : _shooter{shooter}, _humanControl{humanControl} {
+        AddRequirements(&shooter);
+    }
+
+void ShooterCommand::Execute(){
+    if(_humanControl.GetFlywheelDesired()){
+        _shooter.SetFlywheelOutput(.75);
```

Comment:
Need to be able to adjust flywheel speed using the dial. In my code, it was just reading the value off a "joystick," which just represents the dial.